### PR TITLE
Add new docker_registry.server to replace address and port

### PR DESCRIPTION
**Pull request by dkoshkin**
Fixes https://github.com/apprenda/kismatic/issues/837

```
# plan file
docker_registry:                         
  server: registry.example.com:8443        
  CA: /certs/ca.rt    
```

This PR does not remove the fields completely just deprecates them, the user may still use the old fields.

TODO: modify provisioner.

### CODE REVIEW

File: docs/upgrade/v1.6.0/disconnected-upgrade.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ assist you in this task. For more information, see the
 
 ### Existing cluster with registry stood up by KET
 In this scenario, your local registry was stood up by KET and the docker registry
-related fields (`docker_registry.address` and `docker_registry.port`) are not 
+related field (`docker_registry.server`) is not
```
Comment (alexbrand) [inline docs/upgrade/v1.6.0/disconnected-upgrade.md]: I wouldn't update this doc, given that it is specific to version 1.6.0, and this field won't be there in that version.

File: integration/plan_patterns.go [hunk 2]
```diff
@@ -82,8 +81,7 @@ docker:
       block_device: "/dev/xvdb"
       enable_deferred_deletion: false{{end}}
 docker_registry:
-  address: {{.DockerRegistryIP}}
```
Comment (alexbrand) [inline integration/plan_patterns.go]: Will this removal affect upgrade tests where we need to set these values?

Comment (dkoshkin) [reply integration/plan_patterns.go]: I though that at first but the tests only use the registry when doing the upgrade.

**Comment by alexbrand** [comment]
LGTM

**Pull request merged**